cmd/thanos/tools_bucket.go

```diff
@@ -351,7 +351,7 @@ func registerBucketWeb(app extkingpin.AppClause, objStoreConfig *extflag.PathOrC
 		api := v1.NewBlocksAPI(logger, *label, flagsMap)
 
 		// Configure Request Logging for HTTP calls.
-		opts := []logging.Option{logging.WithDecider(func() logging.Decision {
+		opts := []logging.Option{logging.WithDecider(func(_ string, _ error) logging.Decision {
```

Comment:
Same for this, do we want to add logging for this component?

Reply:
Let's start small. I don't think we need until someone asks (: